content/en/docs/concepts/cluster-administration/_index.md

```diff
@@ -52,6 +52,7 @@ Before choosing a guide, here are some considerations:
 ## Managing a cluster
 
 * Learn how to [manage nodes](/docs/concepts/architecture/nodes/).
+  * Read about [cluster autoscaling](/docs/concepts/cluster-administration/cluster-autoscaling/).
```

Comment:
According to me if we use here `[Cluster node autoscaling] ` will be good to new readers to interpret.

Reply:
I think it might make people think we're scaling individual nodes. The rest of the project uses the term “cluster autoscalingg” so we should teach that term, even if we add an explanation.

Reply:
Sounds good, So in `architecture/nodes.md ` file as changed above We can modify it from `cluster node autoscaling` to `cluster autoscaling`. WDYT?

Reply:
I made that change.